### ISSUE TITLE
Могу передаелать в пакет

### ISSUE THREAD

**Issue by PSalador** [tabuna/Vision-Clinic#2]
Могу переделать Vision-Clinic в пакет с комментариями и пошаговым разъяснением.
Сделать?

**Comment by tabuna** [comment]
Это сделало бы меня счастливым к новому году :tada:

Только может лучше не в пакет, а как [руководство](https://github.com/orchidsoftware/platform/blob/master/docs/ru/tutorial_blog.md)

Планировал сделать 3 таких пошаговых статей:
- [Блог](https://github.com/orchidsoftware/platform/blob/master/docs/ru/tutorial_blog.md) (Использование поведение)
- [Клиника](https://github.com/orchidsoftware/platform/blob/master/docs/ru/tutorial_clinic.md) или что то другое (Использование экранов), но я брал от [сюда](https://msdn.microsoft.com/ru-ru/library/ff852064.aspx)
- [Разработка пакета](https://github.com/orchidsoftware/platform/blob/master/docs/ru/tutorial_monitor.md) (Это уже для все возможных вещей которые не были охвачены ранее)

Я уже и заготовку [делал](https://github.com/orchidsoftware/platform/blob/master/docs/ru/tutorial_clinic.md)

**Comment by PSalador** [comment]
> Только может лучше не в пакет, а как руководство

> - Клиника или что то другое (Использование экранов), но я брал от сюда
> - Разработка пакета (Это уже для все возможных вещей которые не были охвачены ранее)

Вот и будет объедение этих пунктов в руководство.
Приведу преимущество пакета для меня как для новичка в Laravel:
1. Всё в одном месте, не забудешь какой файл изменял.
2. Установить пакет легче, не зная git можно скачать как архив.
3. Меньше зависимость от версий Laravel (Сейчас если скачать Vision-Clinic много файлов перезапишут стандартные файлы Laravel) 
4. Меньше конфликтов с другими данными. Например: сейчас в папке [public](https://github.com/tabuna/Vision-Clinic/tree/master/public) осталась ссылка orchid и при установке перезаписывает ссылку созданную командой `php artisan orchid:link` и не дает снова выполнить эту команду.


Так то файлы и содержимое такие же останутся, только пути расположения поменяются.

**Comment by PSalador** [comment]
Так-то хотел свой [пакет ](https://github.com/PSalador/uslugi) расписать, но до экранов всё слишком сложно было, а сейчас он слишком большой становится для расписывания. И ещё смотрю, что у меня структура получается почти одинаковая с Vision-Clinic.
